packages/ragbits-core/docs/how-to/hybrid-search.md

```diff
@@ -0,0 +1,169 @@
+# How to Implement Hybrid Search with Dense and Sparse Embeddings
+
+This guide explains how to implement hybrid search combining both dense and sparse embeddings in Ragbits.
+
+## What is Hybrid Search?
+
+Hybrid search combines multiple search approaches to get the best of both worlds:
+
+- **Dense embeddings**: Good at capturing semantic meaning and contextual relationships
+- **Sparse embeddings**: Excel at lexical matching and keyword preservation
+
+By combining these approaches, you can create more robust search systems that handle both semantic similarity and exact keyword matching.
+
+## Implementing Hybrid Search in Ragbits
+
+Ragbits supports hybrid search through the `HybridSearchVectorStore` class, which can combine results from multiple vector stores.
+
+### Step 1: Create Dense and Sparse Embedders
+
+First, set up both dense and sparse embedders:
+
+```python
+from ragbits.core.embeddings.base import Embedder
+from ragbits.core.embeddings.sparse import SparseEmbedder
+
+# Dense embedder (example using a pre-configured embedder)
+dense_embedder = Embedder.from_config({
+    "type": "ragbits.core.embeddings.openai.OpenAIEmbedder",
```

Comment:
We don't have an `OpenAIEmbedder` class. It is also unclear why use `from_config` in this example when you could instantiate the class directly.